pkg/csource/options.go

```diff
@@ -27,12 +27,14 @@ type Options struct {
 	FaultNth  int  `json:"fault_nth,omitempty"`
 
 	// These options allow for a more fine-tuned control over the generated C code.
-	EnableTun     bool `json:"tun,omitempty"`
-	UseTmpDir     bool `json:"tmpdir,omitempty"`
-	EnableCgroups bool `json:"cgroups,omitempty"`
-	EnableNetdev  bool `json:"netdev,omitempty"`
-	ResetNet      bool `json:"resetnet,omitempty"`
-	HandleSegv    bool `json:"segv,omitempty"`
+	EnableTun           bool `json:"tun,omitempty"`
+	UseTmpDir           bool `json:"tmpdir,omitempty"`
+	EnableCgroups       bool `json:"cgroups,omitempty"`
+	EnableNetdevTypes   bool `json:"netdev_devtypes,omitempty"`
```

Comment:
Let's do `netdev_types`. Shorter, matches the variable name and repeating `dev` twice looks unnecessary.